code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -17,10 +17,10 @@
 	species_clothing_path = 'icons/mob/clothing/species/kepori.dmi'
 	species_eye_path = 'icons/mob/species/kepori/kepori_eyes.dmi'
 	heatmod = 0.67
-	coldmod = 1.5
-	brutemod = 1.5
-	burnmod = 1.5
-	speedmod = -0.25
+	// coldmod = 1.5
+	// brutemod = 1.5
+	// burnmod = 1.5
```

Comment:
```suggestion
	brutemod = 1.25
	burnmod = 1.25
```
Increases effective health from 66 to 80. For something even more slight go with 1.1, which would increase effective health to a little over 90. Speed boosts are noticeable, but I don't think it's -30 health levels of noticeable

Reply:
1. The damage mods just aren't interesting and mess with the way we balance bullet/mob damage. I'd like to come up with something more interesting.
2. The speed boost was always a *lot*, I should have adjusted it when I adjusted config move modifiers but didn't.